src/Raven.Studio/typescript/models/resources/creation/databaseCreationModel.ts

```diff
@@ -154,12 +161,20 @@ class databaseCreationModel {
         dynamicMode: ko.observable<boolean>(true),
         nodes: ko.observableArray<clusterNode>([])
     };
+    
+    sharding = {
+        numberOfShards: ko.observable<number>(2), // ??? 1/2/3 ???
```

Comment:
don't leave comments w/o TODO as it makes almost not possible to find

Reply:
done